snap/snapcraft.yaml

```diff
@@ -221,6 +221,8 @@ parts:
     - -DMULTIPASS_UPSTREAM=origin
     - -DMULTIPASS_ENABLE_FLUTTER_GUI=on
     override-build: |
+      # Bootstrap vcpkg explicitly so we can see the bootstrap log
+      ${CRAFT_PART_SRC}/3rd-party/vcpkg/bootstrap-vcpkg.sh
```

Comment:
Isn't CMake a better place for this though? We can then run on all builds, not just snapcraft. Just need to call the right script depending on the platform.

What I used to debug should be close to what we need, so maybe I push that in another PR. WDYT?

Reply:
Yeah, we could do that, but it does not add much value to local builds since we can poke around freely. For GitHub, I'd rather have it as a dedicated step anyway, which leaves us with Snapcraft. 

I don't mind having it either way, so let me know how you'd prefer it.